eras/conway/test-suite/cardano-ledger-conway-test.cabal

```diff
@@ -42,7 +42,7 @@ library
         cardano-ledger-babbage-test,
         cardano-ledger-binary:{cardano-ledger-binary, testlib} >=1.0,
         cardano-strict-containers,
-        cardano-ledger-conway:{cardano-ledger-conway, testlib} ^>=1.6,
+        cardano-ledger-conway:{cardano-ledger-conway, testlib} ^>=1.7,
```

Comment:
Shouldn't we bump the minor version of this package (cardano-ledger-conway-test) since we updated this bound?

Reply:
Yes, you are right we do need to bump it up, however about an hour ago I merged a PR that already bumps it. So, it just needs a rebase on master.